infrastructure/nomad-configuration/client-instance-user-data.tpl.sh

```diff
@@ -1,4 +1,8 @@
-#!/bin/bash -x
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
```

Comment:
does this boundary get replaced?

Reply:
Nope. I guess I could change that but it seems to work as is. Can you think of anything that's significantly better?

Reply:
Yeah I just thought it looked strange since I'm used to seeing browser generated strings for that value. There is nothing wrong with it as long as there are no collisions. I think it's fine as is, just checking.